src/utils/fetchConfig.ts

```diff
@@ -35,50 +35,50 @@ export async function fetchConfig (): Promise<any> {
 
   const authWebUrl: string = response.data['AUTH_WEB_URL']
   sessionStorage.setItem('AUTH_WEB_URL', authWebUrl)
-  console.log('Set Auth Web URL to: ' + authWebUrl)
```

Comment:
Some were console.log and some were console.info so I changed them all to info for consistency